test/tasks_fine_tuning_task_test.py

```diff
@@ -30,8 +37,11 @@ def _get_pre_train_config(self, head_num_classes=1000):
         return config
 
     def test_build_task(self):
-        config = self._get_fine_tuning_config()
-        task = build_task(config)
+        config = self._get_fine_tuning_config(pretrained_checkpoint=True)
+
+        with mock.patch("classy_vision.tasks.FineTuningTask.set_pretrained_checkpoint"):
+            task = build_task(config)
+
         self.assertIsInstance(task, FineTuningTask)
```

Comment:
Can we retain this test case as is, and then add the new one with `pretrained_checkpoint=True` after this? I want us to test building in both situations